src/test/java/org/sagebionetworks/web/unitclient/widget/entity/team/controller/TeamEditModalWidgetTest.java

```diff
@@ -431,6 +431,16 @@ public void testConfigurePublicCanJoin() {
     verify(mockTeam).setCanRequestMembership(false);
   }
 
+  @Test
+  public void testConfigurePublicCanJoinCanRequestMembershipFalse() {
```

Comment:
All existing tests still pass.  Add test for failure case, when canRequestMembership == false (not null).  This failed before the change, and passes now.